codegen/sdk-codegen/aws-models-test/route-53/route-53-tests.smithy

```diff
@@ -0,0 +1,20 @@
+$version: "1.0"
+
+namespace com.amazonaws.route53
+
+use smithy.test#httpRequestTests
+
+
+apply ListResourceRecordSets @httpRequestTests([
+    {
+        id: "ListResourceRecordSetsTrimHostdZone",
```

Comment:
nit: sp ListResourceRecordSetsTrimHostdZone -> ListResourceRecordSetsTrimHostedZone

Reply:
i'll update and notify rust to update theirs